[feature] add barrak2021co paper

arumoy-shome opened the pull request:
@ckaestne please review.

arumoy-shome committed:
[feature] add barrak2021co paper

ckaestne commented:
Thanks for suggesting it. It was on my reading list for a while.

Would you mind moving it to the end of the section and starting with the sentence "DVC is an open source tool for versioning data and ML workflows. This paper explores how DVC is used in the few open source projects that adopt it." or something similar?

(The paper is interesting, but DVC is really not widely adopted in (open source) practice. The studied systems in the paper even include one of our test projects trying DVC as one of the most active projects :) )

arumoy-shome committed:
[refactor] address comments & summarize findings

1. Move the entry to the bottom of the section
2. Address comment on sentence structure.
3. Add a brief note on the findings.

arumoy-shome commented:
@ckaestne I added a sentence on the findings of the paper as well, let me know what you think.

ckaestne commented:
looks good, thanks!
